common.env

```diff
@@ -6,3 +6,4 @@ CACHE_URL=redis://redis:6379/0
 CELERY_BROKER_URL=redis://redis:6379/1
 SECRET_KEY=changeme
 JWT_VERIFY_EXPIRATION=True
+VATLAYER_ACCESS_KEY=changemetoenablevat
```

Comment:
I think it's ok now. This pattern follows other variables and doesn't mean anything.

Reply:
Usually we are checking if setting is present, rather than if it's valid, or if it means anything
eg in: 
https://github.com/mirumee/saleor/pull/1929/files#diff-8b37ac0a6e8e51f07fe38b86f7d88b06R87
This simple placeholder would enable VAT support even tho it's not valid

Reply:
You're right 👍  I'll fix it.